test/unit/lib/action.test.js

```diff
@@ -233,7 +242,50 @@ describe('lib/action', () => {
 
 			it('clicks the specified element on the page', () => {
 				expect(puppeteer.mockPage.click).toHaveBeenCalledTimes(1);
-				expect(puppeteer.mockPage.click).toHaveBeenCalledWith(clickElementMatches[2]);
+				expect(puppeteer.mockPage.click).toHaveBeenCalledWith(clickElementMatches[3], {});
```

Comment:
Since I've changed the payload above (sorry):

```suggestion
				expect(puppeteer.mockPage.click).toHaveBeenCalledWith(clickElementMatches[3], {
					clickCount: 1
				});
```